pkg/notes/document/template.go

```diff
@@ -63,6 +63,24 @@ filename | sha512 hash
 {{- end -}}
 # Changelog since {{$PreviousRevision}}
 
+{{with .CVEList -}}
+## Important Security Information
+
+This release contains changes that address the following vulnerabilities:
+{{range .}}
+### {{.Title}}
+
+{{.Description}}
+
+__Rating:__ {{.Rating}} — __Score:__ {{.Score}} — __Published:__ {{.Published}}
+
+{{if .LinkedPRs -}}
```

Comment:
I would prefer to just link to the tracking issue (e.g. https://github.com/kubernetes/kubernetes/issues/92914) rather than the specific PRs. The tracking issue should have all the detail needed, and is generally what we consider the source of truth for vulnerability details.

Reply:
:+1:  I've dropped the PRs from the output in favor of a new link to the tracking issue